website/docs/design/options.md

```diff
@@ -13,7 +13,7 @@ Option | Explanation
 `disableFocusFirstItemOnClick` | If specified as `true` the default behavior of focusing the first menu item on click will be disabled. The menu button will instead retain focus.
 
 ```js
-const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(numberOfItems, {
+const { buttonProps, itemProps, isOpen, moveFocus, setIsOpen } = useDropdownMenu(numberOfItems, {
```

Comment:
Also in #283, I plan to remove the variable instantiation entirely, since this page really only focuses on the options (passed as the second argument to `userDropdownMenu()`. In light of that, let's leave the file unchanged in this PR.

```suggestion
const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(numberOfItems, {
```